docs/index.rst

```diff
@@ -17,17 +17,60 @@
 
 
 .. include:: ../README.rst
+   :end-before: Usage
+
+For more information about the Renga API `see its documentation
+<https://renga.readthedocs.org/latest/developer/index.html>`_.
+
+Use the Renga command line
+--------------------------
+
+Interaction with the platform can take place via the command-line
+interface (CLI).
+
+Start by creating for folder where you want to keep your Renga project:
```

Comment:
This is now duplicated in the README and here -- is that intentional?

Reply:
we should find a way to not duplicate this text in the "first steps" in the main `renga` repo

Reply:
I would remove it from README and just point to docs directly ...